Intersect.Editor/Forms/Editors/Events/Event Commands/EventCommand_ConditionalBranch.cs

```diff
@@ -1271,6 +1293,11 @@ private void SetupFormValues(MapZoneTypeIs condition)
             }
         }
 
+        private void SetupFormValues(CheckEquippedSlot condition)
+        {
+            cmbCheckEquippedSlot.SelectedIndex = condition.SlotIndex;
```

Comment:
Instead of `condition.SlotIndex` it would be set to `Options.EquipmentSlots.IndexOf(condition.Name)`